gpt4all-backend/llamamodel.cpp

```diff
@@ -28,6 +25,26 @@
 #include <thread>
 #include <unordered_set>
 
+#include <llama.h>
+#include <ggml.h>
+
+namespace {
+const char *modelType_ = "LLaMa";
```

Comment:
Any reason for the capitalization here?

Reply:
Actually you are correct, the capitalization was wrong! Fixed now.